packages/rich-text-links/bin/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "target": "es2015"
+    "target": "es2017"
```

Comment:
No clue why this is needed now, but it it's fine with the customer JS versions you're targeting for it's fine with me 🙂

Reply:
This is meant to reflect the `ts-node`/`node` versioning used in development, which should include full es2017 support for everything other than more niche features like `SharedArrayBuffer` and atomics. In particular though, I intended to keep the compiler from complaining about `Object.entries`.